pex/bin/pex.py

```diff
@@ -159,7 +159,7 @@ def configure_clp_pex_resolution(parser, builder):
   group.add_option(
       '--cache-dir',
       dest='cache_dir',
-      default=os.path.expanduser('~/.pex/build'),
+      default='{pex_root}/build',
```

Comment:
the help string for this option references the default value used here, so it seems to me that the help output for this option would go from:

```
    --interpreter-cache-dir=INTERPRETER_CACHE_DIR
                        The interpreter cache to use for keeping track of
                        interpreter dependencies for the pex tool. [Default:
                        /Users/kwilson/.pex/interpreters]
```

to this:

```
    --interpreter-cache-dir=INTERPRETER_CACHE_DIR
                        The interpreter cache to use for keeping track of
                        interpreter dependencies for the pex tool. [Default:
                        {pex_root}/interpreters]
```

how about updating the help strings both here and for `--interpreter-cache-dir` with the literal defaults (e.g. the `~/.pex/...` path) so it's more user friendly?